securedrop/journalist_app/admin.py

```diff
@@ -172,6 +176,24 @@ def edit_user(user_id):
                 else:
                     user.username = new_username
 
+            try:
+                first_name = request.form['first_name']
+                Journalist.check_name_acceptable(first_name)
+                if first_name != user.first_name:
```

Comment:
These `if` checks are unnecessary since SQLAlchemy uses listeners generated by the metaclass to check for changes to the object.